Misc/NEWS.d/next/Library/2024-02-09-07-20-16.gh-issue-115165.yfJLXA.rst

```diff
@@ -0,0 +1,3 @@
+``typing.Annotated`` ignores exceptions when setting the ``__orig_class__``
```

Comment:
```suggestion
``typing.Annotated`` now ignores all exceptions when setting the ``__orig_class__``
```
Make it a little more clear what changed

Reply:
addressed in 1d7e6e5b1a750793d09e5f8922da573c4a1b5c53